bin/strata-cli/src/cmd/recover.rs

```diff
@@ -76,8 +79,17 @@ pub async fn recover(args: RecoverArgs, seed: Seed, settings: Settings) {
             continue;
         }
 
+        // F***! BDK 1.0 removed the `insert_tx` method, so we have to do this the hard way
+        // Taken from https://github.com/bitcoindevkit/bdk/pull/1658/commits/3135e291d777d474ab4b76de36d43a96ff104a3c
         recovery_wallet.transactions().for_each(|tx| {
-            l1w.insert_tx(tx.tx_node.tx);
+            l1w.apply_update(Update {
+                tx_update: TxUpdate {
+                    txs: vec![tx.tx_node.tx],
+                    ..Default::default()
+                },
+                ..Default::default()
+            })
+            .expect("could not insert transaction");
```

Comment:
Any ideas here @Zk2u?

Reply:
IIRC you can use the [`apply_unconfirmed_txs`](https://docs.rs/bdk_wallet/latest/bdk_wallet/struct.Wallet.html#method.apply_unconfirmed_txs).